dotnet/src/Microsoft.AutoGen/Agents/Agents/IOAgent/ConsoleAgent/IHandleConsole.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.AutoGen.Abstractions;
+
+namespace Microsoft.AutoGen.Agents
+{
+    public interface IHandleConsole : IHandle<Output>, IHandle<Input>
+    {
+        string Route { get; }
+        AgentId AgentId { get; }
+        ValueTask PublishEvent(CloudEvent item);
```

Comment:
Still need those methods? I thought they are already defined in `IAgentBase` interface

Reply:
yes, needed for the publishevent to function from here - consider this a :partial agent. opportunity for refactor to a smaller common base or to have this interface inherit from IAgentBase